doc/source/io.rst

```diff
@@ -105,6 +105,30 @@ or a slice object.
         rows=slice(10, 20),
     )
 
+Field/Column Filters
+^^^^^^^^^^^^^^^^^^^^
+
+Load in a subset of fields from the file:
+
+.. note:: Requires Fiona 1.8+
+
+.. code-block:: python
+
+    gdf = geopandas.read_file(
+        geopandas.datasets.get_path("naturalearth_lowres"),
+        ignore_fields=["iso_a3", "gdp_md_est"],
+    )
+
+Skip loading geometry from the file:
```

Comment:
Probably a good idea to note that this returns a `DataFrame` instead of a `GeoDataFrame`.  The `pdf` vs `gdf` is a good hint, but more explicit would probably be good.

Reply:
Added a note for that. Definitely a good call out to prevent confusion.